src/color.cc

```diff
@@ -15,23 +17,249 @@
 #define WHITESPACE \
   while (' ' == *str) ++str;
 
+#define WHITESPACE_OR_COMMA \
+  while (' ' == *str || ',' == *str) ++str;
+
+
+/*
+ * Parse integer value
+ */
+
+template <typename parsed_t>
+static bool parseInteger(const char** pStr, parsed_t *pParsed)
+{
+  parsed_t& c = *pParsed;
+  const char*& str = *pStr;
+  int8_t sign=1;
+
+  c = 0;
+  if (*str == '-')
+  {
```

Comment:
lots of inconsistencies here with brace style, some of them are with a newline and some are not; have a look through the rest of the project to find a happy place with consistency